_data/projects.yml

```diff
@@ -6,6 +6,8 @@
   link: https://editor.uclaacm.com
   repo: uclaacm/TeachLAFrontend
   lead:
+    - name: Timothy
+      hash: tpoon
     - name: Leo
```

Comment:
Can remove since they're underneath the previous boards section of the teams page.